Makefile

```diff
@@ -0,0 +1,136 @@
+# Hiredis-cluster Makefile
+# Copyright (C) 2010-2011 Salvatore Sanfilippo <antirez at gmail dot com>
+# Copyright (C) 2010-2011 Pieter Noordhuis <pcnoordhuis at gmail dot com>
+# This file is released under the BSD license, see the COPYING file
```

Comment:
Is it copied from somewhere a place and not changed since 2011? It's hard to know since Hiredis-cluster is new... Perhaps clarify where it was copied from or add your own copyright line.